backend/tests/Notifo.Domain.Tests/Integrations/Discord/DiscordTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Notifo.Domain.Integrations.Discord;
+public sealed class DiscordTests
+{
+    [Fact]
+    public void Should_get_integration()
+    {
+        var client = CreateClient();
```

Comment:
You can also create the client in the constructor and add it as a field.

See: https://github.com/notifo-io/notifo/blob/main/backend/tests/Notifo.Domain.Tests/Counter/CounterCollectorTests.cs#L17 

Usually I call it "sut" (System under Test)